platypus-core/src/test/java/com/platypus/crw/udp/UdpVehicleServerTest.java

```diff
@@ -45,6 +46,8 @@
     
     UdpVehicleService service;
     SimpleBoatSimulator sbs;
+    int port;
+    InetSocketAddress localhost;
```

Comment:
Rename this `serviceAddress` because it's actually "localhost:PORT" which is technically more than a hostname.